source/fundamentals/aggregation/aggregation-expression-operations.txt

```diff
@@ -157,5 +157,75 @@ the Query API:
                                       then: "extreme temperature",
                                       else: "no advisory" } } } }
 
+Comparison Operations
+~~~~~~~~~~~~~~~~~~~~~
+
+You can use comparison operations to compare two different values. You can
```

Comment:
S: Can either the first or second sentence be rephrased to avoid two consecutive "You can..." sentences?